### ISSUE TITLE
Notice: Undefined index: activated_status

### ISSUE THREAD

**Issue by emanuele45** [elkarte/Elkarte#3005]
```
http://www.elkarte.net/community/index.php?action=admin;area=viewmembers;sa=approve;type=activate
```

Notice: Undefined index: activated_status
File: sources/subs/Members.subs.php
Line: 2010

**Comment by emanuele45** [comment]
Likely related to #3003 and #3004

**Comment by emanuele45** [comment]
Fixed in 2b8717894ef9d56ce6a8a60ea5152949ac64155c
